### ISSUE TITLE
Check arrays

### ISSUE THREAD

**Issue by fabien88** [hpersson/eslint-plugin-jsx-expressions#7]
It seems that arrays are not properly checked :

```
const myArray = [''];
...
{myArray[0] && ...}
```

doesn't throw any error.

BTW, Thanks for this great plugin !
